jwst/model_blender/rules.py

```diff
@@ -0,0 +1,112 @@
+import numpy as np
+
+
+__all__ = ['RULE_FUNCTIONS', 'AttributeBlender', 'make_blender']
+
+
+def _multi(vals):
+    """
+    This will either return the common value from a list of identical values
+    or 'MULTIPLE'
+    """
+    uniq_vals = list(set(vals))
+    num_vals = len(uniq_vals)
+    if num_vals == 0:
+        return None
+    if num_vals == 1:
+        return uniq_vals[0]
+    if num_vals > 1:
+        return "MULTIPLE"
+
+
+RULE_FUNCTIONS = {
+    'multi': _multi,
+    'mean': np.mean,
+    'sum': np.sum,
+    'max': np.max,
+    'min': np.min,
+
+     # retained date/time names for backwards compatibility
+     # as these all assume ISO8601 format the lexical and
+     # chronological sorting match
+    'mintime': min,
+    'maxtime': max,
+    'mindate': min,
+    'maxdate': max,
+    'mindatetime': min,
+    'maxdatetime': max
+}
+"""
+Mapping of rule names to functions.
+
+Used for `make_blender`.
+
```

Comment:
I know it seems a little redundant, since they're listed above, but can you add the list of not-deprecated rules here?  In the built docs, the deprecated ones are easy to see but the supported ones are not.

Reply:
I noticed that. I'm not sure why sphinx rendered those in a weird way. I'd rather not duplicate the code in the docs (so they can automatically stay in sync). Although I'd also like to not fight with sphinx for an unknown amount of time to fix the rendering. If the poor rendering is a "deal breaker" I'll duplicate the entries. Let me know either way.